it-tests/PropertyPanelLoading.test.ts

```diff
@@ -22,7 +22,7 @@ describe('Property panel loading test', function () {
     await editorView.closeAllEditors();
   });
 
-  it('Open "choice.camel.yaml" file and check property panel is loading and closing', async function () {
+  it.only('Open "choice.camel.yaml" file and check property panel is loading and closing', async function () {
```

Comment:
.only - needs to be removed before merge

Reply:
:face_with_spiral_eyes: